### ISSUE TITLE
com.jcraft.jsch.JSchException: Auth fail: Auth fail

### ISSUE THREAD

**Issue by jicai** [eclipse-ee4j/glassfish#24479]
## Environment Details
* GlassFish Version (and build number): Eclipse GlassFish 7.0.5 (branch: master, commit: 7b49f4eaf669e9a3b4f4648254f9016872455236, timestamp: 2023-04-02T15:44:03Z)
* JDK version:Java HotSpot(TM) 64-Bit Server VM (build 17.0.7+8-LTS-224, mixed mode, sharing)
* OS:rhel-9.2-x86_64 Linux version 5.14.0-284.11.1.el9_2.x86_64
* OpenSSH：OpenSSH_8.7p1, OpenSSL 3.0.7 1 Nov 2022

----------
I want to form a cluster with four machines：RedHat921、RedHat922、RedHat923、RedHat924。RedHat921 AS DAS，RedHat922、RedHat923、RedHat924 as nodes.

I have installed glassfish 7.0.5 on every machine(just unzip)

I executed the following command on machine RedHat921：
`create-node-ssh --nodehost RedHat922 --install true node112`

I got the exception as flows：
```
Could not connect to host RedHat922 using SSH.
com.jcraft.jsch.JSchException: Auth fail
Auth fail
```

The infometion in server.log :
`2023-06-25T17:56:38.502423+08:00] [GF 7.0.5] [WARNING] [] [jakarta.enterprise.system.core] [tid: _ThreadID=160 _ThreadName=admin-listener(21)] [levelValue: 900] [[
  Could not connect to host RedHat922 using SSH.: com.jcraft.jsch.JSchException: Auth fail: Auth fail: host=RedHat922 port=22 user=jensen password=null keyFile=/home/jensen/.ssh/id_rsa keyPassPhrase=null authType=null knownHostFile=/home/jensen/.ssh/known_hosts]]`

Can tell me the reason or give me some advice? thanks

**Comment by avpinchuk** [comment]
@jicai, please try with newly released GF 7.0.6

**Comment by jicai** [comment]
@OndroMih @avpinchuk   When I switched to version 7.0.6, everything was fine. Thank you.
